src/teleop/gui/src/components/AutonModeCheckbox.vue

```diff
@@ -1,68 +1,67 @@
 <template>
   <div class="wrap-button">
-    <button v-bind:class="[this.color]" v-on:click='toggleAndEmit()'>
-      <span class='white-text'>{{name}}: <br>{{active ? '\u2611' : '\u2610'}}</span>
+    <button :class="[color]" @click="toggleAndEmit()">
+      <span class="white-text"
+        >{{ name }}: <br />{{ active ? "\u2611" : "\u2610" }}</span
+      >
```

Comment:
Kinda in disbelief this is how it would want this section formatted

Reply:
https://prettier.io/blog/2018/11/07/1.15.0.html#whitespace-sensitive-formatting
It's due to this rule according to stackoverflow